unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTF/AssetPromise_GLTF.cs

```diff
@@ -12,13 +12,27 @@ public class AssetPromise_GLTF : AssetPromise<Asset_GLTF>
         public string url { get; private set; }
 
         GLTFComponent gltfComponent = null;
+        IWebRequestController webRequestController = null;
+
         object id = null;
 
+        public AssetPromise_GLTF(string url, IWebRequestController webRequestController)
```

Comment:
Made the web request work through dependency injection to be able to mock promises without having to initialize the whole `Environment`.  Also you don't have to pass the annoying `ContentProvider`.